samples/nextjs-template/src/app/json/[...slug]/page.tsx

```diff
@@ -36,7 +32,7 @@ export default async function Page({ params }: Props) {
   // Note: this is shown for demo purposes.
   // `fetchContentType` and `createQuery` are not needed
   const contentType = await client
-    .fetchContentType(getFilterFromPath(path))
+    .fetchContentType(path)
     .catch(handleGraphErrors);
   const query = createQuery(contentType);
```

Comment:
Not part of this PR, but this is probably more confusing than helping if we're not using/executing `query`.